source-code/website/src/pages/index/Header.tsx

```diff
@@ -1,189 +1,108 @@
-									</svg>
-								</a>
-								<a
-									href="https://github.com/inlang/inlang"
-									class="link-primary  flex space-x-1"
-								>
-									<span class="">GitHub</span>
-									<svg
-										class="h-6 w-6"
-										fill="currentColor"
-										viewBox="0 0 24 24"
-										aria-hidden="true"
+					<div class="z-10 p-2 grid grid-cols-2 w-full items-center">
+						<div class="hidden md:flex justify-start space-x-6">
+							<For each={links}>
+								{(link) => (
+									<a class="link-primary" href={link.href}>
+										{link.name}
+									</a>
+								)}
+							</For>
+						</div>
+						<div class="hidden md:flex justify-end space-x-6">
+							<For each={socialMediaLinks}>
+								{(link) => (
+									<a
+										class="link-primary flex space-x-2 items-center"
+										href={link.href}
 									>
-										<path
-											fill-rule="evenodd"
-											d="M12 2C6.477 2 2 6.484 2 12.017c0 4.425 2.865 8.18 6.839 9.504.5.092.682-.217.682-.483 0-.237-.008-.868-.013-1.703-2.782.605-3.369-1.343-3.369-1.343-.454-1.158-1.11-1.466-1.11-1.466-.908-.62.069-.608.069-.608 1.003.07 1.531 1.032 1.531 1.032.892 1.53 2.341 1.088 2.91.832.092-.647.35-1.088.636-1.338-2.22-.253-4.555-1.113-4.555-4.951 0-1.093.39-1.988 1.029-2.688-.103-.253-.446-1.272.098-2.65 0 0 .84-.27 2.75 1.026A9.564 9.564 0 0112 6.844c.85.004 1.705.115 2.504.337 1.909-1.296 2.747-1.027 2.747-1.027.546 1.379.202 2.398.1 2.651.64.7 1.028 1.595 1.028 2.688 0 3.848-2.339 4.695-4.566 4.943.359.309.678.92.678 1.855 0 1.338-.012 2.419-.012 2.747 0 .268.18.58.688.482A10.019 10.019 0 0022 12.017C22 6.484 17.522 2 12 2z"
-											clip-rule="evenodd"
-										/>
-									</svg>
-								</a>
-							</div>
-						</Show>
+										<link.Icon></link.Icon>
+										<span>{link.name}</span>
+									</a>
+								)}
+							</For>
+						</div>
 					</div>
-
-					{/* end icon social media   */}
-
-					<div class=" md:hidden z-50">
-						<Show
-							when={show() == false}
-							fallback={
-								<button
-									onClick={(e) => setShow(false)}
-									type="button"
-									class="inline-flex items-center justify-center  bg-background p-2 text-primary "
-									aria-expanded="false"
-								>
-									<span class="sr-only">Close menu</span>
-									{/* <!-- Heroicon name: outline/x-mark --> */}
-									<svg
-										class="h-6 w-6"
-										xmlns="http://www.w3.org/2000/svg"
-										fill="none"
-										viewBox="0 0 24 24"
-										stroke-width="1.5"
-										stroke="currentColor"
-										aria-hidden="true"
-									>
-										<path
-											stroke-linecap="round"
-											stroke-linejoin="round"
-											d="M6 18L18 6M6 6l12 12"
-										/>
-									</svg>
-								</button>
-							}
+					<div class="md:hidden z-50">
+						<button
+							onClick={() => setMobileMenuIsOpen(!mobileMenuIsOpen())}
```

Comment:
here i am using re-using the markup with dynamic logic. the requirement to have a <Show> component vanished.